wdl-format/tests/format/ENCODE-DCC_chip-seq-pipeline/source.formatted.wdl

```diff
@@ -0,0 +1,3586 @@
+        Boolean enable_jsd = true
+        Boolean enable_gc_bias = true
+
+        # group: alignment
+        String aligner = "bowtie2"
+        File? custom_align_py
+        Boolean use_bwa_mem_for_pe = false
+        Int bwa_mem_read_len_limit = 70
+        Boolean use_bowtie2_local_mode = false
+        Int crop_length = 0
+        Int crop_length_tol = 2
+        String trimmomatic_phred_score_format = "auto"
+        Int xcor_trim_bp = 50
+        Boolean use_filt_pe_ta_for_xcor = false
+        String dup_marker = "picard"
+        Boolean no_dup_removal = false
+        Int mapq_thresh = 30
+        Array[String] filter_chrs = [
+        ]
+        Int subsample_reads = 0
+        Int ctl_subsample_reads = 0
+        Int xcor_subsample_reads = 15000000
+        Int xcor_exclusion_range_min = -500
+        Int? xcor_exclusion_range_max
+        Int pseudoreplication_random_seed = 0
+
+        # group: peak_calling
+        Int ctl_depth_limit = 200000000
+        Float exp_ctl_depth_ratio_limit = 5.0
+        Array[Int?] fraglen = [
+        ]
+        String? peak_caller
+        Boolean always_use_pooled_ctl = true
+        Float ctl_depth_ratio = 1.2
+        Int? cap_num_peak
+        Float pval_thresh = 0.01
+        Float fdr_thresh = 0.01
+        Float idr_thresh = 0.05
+
+        # group: resource_parameter
+        Int align_cpu = 6
+        Float align_bowtie2_mem_factor = 0.15
+        Float align_bwa_mem_factor = 1.0
+        Int align_time_hr = 48
+        Float align_bowtie2_disk_factor = 8.0
+        Float align_bwa_disk_factor = 8.0
+
+        Int filter_cpu = 4
+        Float filter_mem_factor = 0.4
+        Int filter_time_hr = 24
+        Float filter_disk_factor = 8.0
+
+        Int bam2ta_cpu = 2
+        Float bam2ta_mem_factor = 0.35
+        Int bam2ta_time_hr = 6
+        Float bam2ta_disk_factor = 4.0
+
+        Float spr_mem_factor = 20.0
+        Float spr_disk_factor = 30.0
+
+        Int jsd_cpu = 4
+        Float jsd_mem_factor = 0.1
+        Int jsd_time_hr = 6
+        Float jsd_disk_factor = 2.0
+
+        Int xcor_cpu = 2
+        Float xcor_mem_factor = 1.0
+        Int xcor_time_hr = 24
+        Float xcor_disk_factor = 4.5
+
+        Float subsample_ctl_mem_factor = 22.0
+        Float subsample_ctl_disk_factor = 15.0
+
+        Float macs2_signal_track_mem_factor = 12.0
+        Int macs2_signal_track_time_hr = 24
+        Float macs2_signal_track_disk_factor = 80.0
+
+        Int call_peak_cpu = 6
+        Float call_peak_spp_mem_factor = 5.0
+        Float call_peak_macs2_mem_factor = 5.0
+        Int call_peak_time_hr = 72
+        Float call_peak_spp_disk_factor = 5.0
+        Float call_peak_macs2_disk_factor = 30.0
+
+        String? align_trimmomatic_java_heap
+        String? filter_picard_java_heap
+        String? gc_bias_picard_java_heap
+    }
+
+    String pipeline_ver = "v2.2.2"
+    RuntimeEnvironment runtime_environment = {
+        "docker": docker,
```

Comment:
Does it seem weird that we require spacing around the `=` assignment operator, but not the `:` operator? I don't think people generally precede `:` with whitespace.

Reply:
O do not like that idea 😅 
I think the current handling is correct (no space on the LHS, one space on the RHS)